libs/datasets/sources/hsa_population.py

```diff
@@ -0,0 +1,65 @@
+from functools import lru_cache
+
+from datapublic.common_fields import CommonFields
+from libs.datasets import data_source
+from libs.datasets import timeseries
+from libs.pipeline import Region
+from libs.datasets.dataset_utils import AggregationLevel
+from libs.datasets.statistical_areas import CountyToHSAAggregator
+from libs.datasets.sources.fips_population import FIPSPopulation
+
+
+def get_location_level(location_id):
+    location = Region.from_location_id(location_id)
+    # TODO(sean): Weed out locations whose levels cant be determined.
+    # In this case it's fine because we only want to select counties.
```

Comment:
Just for my own understanding, do we have any idea what locations are triggering this?  Or that's what this comment is saying we need to figure out?  I'm fine with leaving this as a TODO. Just checking.

Reply:
I took a look and the two FIPS throwing this are `iso2:us#iso2:us-dc#fips:11999` and `iso2:us#iso2:us-fl#fips:12999`, neither of which are valid locations so I'm not sure what they represent. 

Somewhat unrelated (it wasn't breaking anything here), I realized that I didn't add implementation for  `Region.level` for HSA locations, so I've added that in a01d4a78795e22c05cc217cee9f57da8587a2b7b, though this has no effect on the rest of this PR.